
#spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
spring.datasource.tomcat.max-wait=60000


# ===============================
# = JPA / HIBERNATE
# ===============================
spring.jpa.show-sql=true
spring.jpa.hibernate.ddl-auto=create
spring.jpa.generate-ddl=true
spring.jpa.properties.hibernate.enable_lazy_load_no_trans=true
spring.jpa.database-platform=org.hibernate.dialect.MySQL55Dialect

#spring.datasource.url=jdbc:mysql://localhost:3306/explorer?useUnicode=yes&characterEncoding=UTF-8
#spring.datasource.username=root
#spring.datasource.password=root
#
#spring.datasource.url=jdbc:mysql://35.247.104.62:3306/coxplore
#spring.datasource.username=root
#spring.datasource.password=root

spring.datasource.url=jdbc:mysql://35.247.104.62/contact
spring.datasource.username=root
spring.datasource.password=root
## So app starts despite "table already exists" errors.
#spring.datasource.continue-on-error=true
#
## Enforces database initialization
#spring.datasource.initialization-mode=always

